openpype/hosts/traypublisher/plugins/publish/collect_simple_instances.py

```diff
@@ -4,9 +4,12 @@
 
 import clique
 import pyblish.api
+from openpype.pipeline import publish, legacy_io
 
 
-class CollectSettingsSimpleInstances(pyblish.api.InstancePlugin):
+class CollectSettingsSimpleInstances(
+    pyblish.api.InstancePlugin, publish.ColormanagedPyblishPluginMixin
```

Comment:
Why?

Reply:
Because we need to collect the colorspace data in order to run certain plugins (i.e., transcode).